tests/test-team/helpers/database-tablename-helper.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-use-before-define */
+import {
```

Comment:
this could probably be done on playwright setup and put into env vars?

Reply:
done.

Reply:
I have no strong feelings on how this is done 🦖